packages/nimble-components/src/table/components/cell/template.ts

```diff
@@ -1,9 +1,9 @@
-import { html } from '@microsoft/fast-element';
+import { html, ref } from '@microsoft/fast-element';
 import type { TableCell } from '.';
 
 // prettier-ignore
 export const template = html<TableCell>`
     <template role="cell">
-        ${x => x.data}
+        <div ${ref('cellContainer')} class="cell-container"></div>
```

Comment:
I think when we have other content in a cell, such as an action menu, it won't go in this div. Maybe calling it `cellContentContainer` with a class of `cell-content-container` would be more appropriate.

Reply:
Updated.